src/main/java/org/frc5687/steamworks/protobot/commands/DriveWith2Joysticks.java

```diff
@@ -34,7 +34,12 @@ protected void initialize() {
      * @see edu.wpi.first.wpilibj.command.Command#execute()
      */
     protected void execute() {
-        driveTrain.tankDrive(oi.getLeftSpeed(), oi.getRightSpeed());
+        if (oi.getLeftTrigger()){
+            driveTrain.tankDrive(oi.getLeftSpeed(), oi.getRightSpeed());
```

Comment:
This is confusing, possibly because of the method name. The way it reads now suggests that tank drive is only enabled when the left trigger is actuated. Ideally, the one-stick logic would be in the "if" block, since it is the special case we are looking for. The else block would have the two-stick logic, since that is the default.